ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,942 @@
+|Two invocations under transaction context stared at the `server1` (caller) side.
+The stateless bean is invoked at the remote side.
+The EJB remote call is constructed from the information defined directly
+in the application source code.
+The remote invocation is run, in difference to other calls of this quickstarts, via EJB over HTTP protocol.
+|The returned hostnames have to be the same.
+
+|__http://localhost:8080/client/remote-outbound-notx-stateful__
+|Two invocations under transaction context stared at the `server1` (caller) side.
+Both calls are directed to the same stateful bean on the remote server because
+the stateful bean invocations are sticky by default.
+The EJB remote call is constructed from the configuration of `remote-outboud-connection`.
+|The returned hostnames have to be the same.
+
+|__http://localhost:8080/client/remote-outbound-fail-stateless__
+|An invocation under transaction context stared at the `server1` (caller) side.
+The call goes to one of the remote servers where error happens during transaction processing.
+But the failure happens when two-phase commit decided about commit the work.
+The observer can see no error &ndash; the remote call finishes with success.
+Later it's responsibility of the recovery manager to finish the work.
+|When the recovery manager finishes the work all the transaction resources are committed.
+
+|===
+
+=== Remote call failure ivocation
+
+Let's put some more details for the failure case, when __http://localhost:8080/client/remote-outbound-fail-stateless__
+is invoked.
+
+As stated above the ivocation fails. This failure simulates a intermittent network error
+at time the transaction two-phase commit protocol
+already decided that the work has to be committed. The observer is not informed
+about the intermittent failure as it's responsibility of recovery manager to finish
+all the work.
+
+The work, which has to be finished by recovery, is consisted of committing two XAResources
+which were part of the business method at the caller side of artifact `server.war`.
+First is data insertion to a database. Second is a testing XAResource which does
+no real work but it's capable to inform us if it was committed.
+You can ask the server about the number succesful of commits of the testing XAResource
+by invoking REST endpoint `http://localhost:8180/server/commits`.
+
+The http://jbossts.blogspot.com/2018/01/narayana-periodic-recovery-of-xa.html[recovery manager]
+normally executes the recovery processing in periodically every 2 minutes.
+When the recovery process is started the resources at the remote server (on the callee side)
+are committed.
+
+You may speed up the process and invoke the recovery process manually by accessing
+the port where recovery manager listener listens at. The recovery listener was enabled
+for this purpose by cli command, see `${PATH_TO_QUICKSTART_DIR}/ejb-txn-remote-call/client/extensions/remote-configuration.cli`.
```

Comment:
This cli command does not work

Reply:
I also tried the telnet command below. I got no feedback, what was meant to happen?

Reply:
ok, I need to check.

Reply:
That was not meant that the command should be executed, just that it was already executed. The `telnet` command works, I assume just it's not obvious what to do with it.

I rephrased the whole section. Hopefully it will be more comprehensible now. Let me know.